features/other_b/skin.feature

```diff
@@ -67,6 +73,33 @@ Feature: creating and editing skins
   When I am on skin's new page
     Then I should see "Sorry, you don't have permission"
 
+  Scenario: An admin can reject a skin
```

Comment:
Can you make this "An admin can reject and unreject a skin" so we know it covers both actions?